apolloschurchapp/src/tabs/home/index.js

```diff
@@ -1,3 +1,16 @@
+import React from 'react';
+import { createNativeStackNavigator } from 'react-native-screens/native-stack';
+
 import Home from './Home';
+import Search from './Search';
+
+const { Navigator, Screen } = createNativeStackNavigator();
+
+const HomeNavigator = () => (
+  <Navigator initialRouteName="Home">
+    <Screen component={Home} name="Home" options={{ headerShown: false }} />
+    <Screen component={Search} name="Search" options={{ headerShown: false }} />
```

Comment:
iirc the search screen needs to be moved to the root modal and this navigator is no longer needed
https://github.com/NewSpring/corinth/commit/942b67ec1655b6549088980d999352d8b4ae8401

(note, search is named "discover on NS")

Reply:
that was because the one in NS is a modal, this is just a stack native nav and having it here seems like it makes more sense to me because it's on the home screen. this is how I did LH.

Reply:
Ah, that makes more sense. @conrad-vanl do you care if the default for the search screen is a `modal` or a `stack`?